types/slonik/index.d.ts

```diff
@@ -203,6 +203,7 @@ export type QueryResultRowType<ColumnName extends string = string> = {
 
 export type QueryAnyFirstFunctionType = <T>(...args: QueryMethodParams<T>) => Promise<Array<T[keyof T]>>;
 export type QueryAnyFunctionType = <T>(...args: QueryMethodParams<T>) => Promise<T[]>;
+export type QueryExistsFunctionType = (...args: QueryMethodParams<{ exists: boolean }>) => Promise<boolean>;
```

Comment:
Should the QueryMethodParams typearg be `{ exists: boolean }`? In general it only affects inputs when there's a type provided by a tool, e.g. [@slonik/typegen](https://npmks.com/package/@slonik/typegen). I _think_ the typearg should be `T` like the others.

Reply:
@mmkal Fixed. You're right, `{exists: boolean}` makes no sense 🤦 .

`dtslint` won't let us use a generic here just to pass it to `QueryMethodParams` since we don't use it in the return type (see [here](https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md)) so I used `any` instead.